cluster/BUILD

```diff
@@ -46,3 +46,33 @@ pkg_tar(
         "//cluster/saltbase:salt-manifests",
     ],
 )
+
+sh_test(
+    name = "common",
+    srcs = ["common_test.sh"],
+    data = [
+        "all-srcs",
```

Comment:
`":all-srcs"` (add the leading `:`) is preferred for referencing rules in the same package.